WNPRC_Purchasing/resources/queries/ehr_purchasing/userAccountsAssocFiltered.sql

```diff
@@ -0,0 +1,25 @@
+SELECT *
+FROM (
+
+--    associate all accounts to the user if accessToAllAccounts flag is set to true
+         SELECT a.rowid AS rowid,
+                a.alias AS account,
+                u.userId
+         FROM ehr_billingLinked.aliases a,
+              (SELECT ua.userId,
+                      ua.account
+               FROM ehr_purchasing.userAccountAssociations ua
+               WHERE ua.accessToAllAccounts = true
+                 AND ua.account IS NULL) u
+
+         UNION ALL
```

Comment:
I think you want UNION instead of UNION ALL here in case a user is configured with double-access (both to individual accounts and all accounts).

Reply:
Yup! now changed!